SPECS/docbook-style-xsl/docbook-style-xsl.spec

```diff
@@ -24,6 +26,12 @@ allowing you to utilize transformations already written for that standard.
 
 %prep
 %setup -q -n docbook-xsl-%{version}
+# CVE-2022-34169: xalan 2.7.2 has security issue that is solved by 2.7.3,
+# so replace the embedded jar files in docbook-xsl release before continuing
+mkdir ./CVE-2022-34169
+tar -xf %{SOURCE1} -C ./CVE-2022-34169
+mv ./CVE-2022-34169/xalan-j_2_7_3/*.jar ./tools/lib/.
```

Comment:
That directory seems to contain the following JARs:
```bash
ll downloaded_sources/xalan-j/xalan-j_2_7_3/*.jar  
-rw-r--r-- 1 pawel pawel 277K Mar 25  2023 downloaded_sources/xalan-j/xalan-j_2_7_3/serializer.jar
-rw-r--r-- 1 pawel pawel 3.3M Mar 25  2023 downloaded_sources/xalan-j/xalan-j_2_7_3/xalan.jar
-rw-r--r-- 1 pawel pawel 1.4M Mar 25  2023 downloaded_sources/xalan-j/xalan-j_2_7_3/xercesImpl.jar
-rw-r--r-- 1 pawel pawel 224K Mar 25  2023 downloaded_sources/xalan-j/xalan-j_2_7_3/xml-apis.jar
```

The package, however, seems to only provide `xalan.jar` and `xercesImpl.jar`, so we're moving a bit too much right now.

Reply:
yeah, i noticed that but thought it was safer to keep all of them in case 2.7.3 introduced a need for the extra jar.

Reply:
I think you're right. I just run `jdeps` and it seems to confirm the dependency:
```bash
jdeps -s downloaded_sources/xalan-j/xalan-j_2_7_3/*.jar
(...)
xalan.jar -> downloaded_sources/xalan-j/xalan-j_2_7_3/serializer.jar
xalan.jar -> downloaded_sources/xalan-j/xalan-j_2_7_3/xercesImpl.jar
xalan.jar -> downloaded_sources/xalan-j/xalan-j_2_7_3/xml-apis.jar
...
```